airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityImpl.java

```diff
@@ -139,7 +139,13 @@ public void jobFailure(final JobFailureInput input) {
   public void attemptFailure(final AttemptFailureInput input) {
     try {
       jobPersistence.failAttempt(input.getJobId(), input.getAttemptId());
-      final Job job = jobPersistence.getJob(input.getJobId());
+
+      if (input.getStandardSyncOutput() != null) {
+        final JobOutput jobOutput = new JobOutput().withSync(input.getStandardSyncOutput());
+        jobPersistence.writeOutput(input.getJobId(), input.getAttemptId(), jobOutput);
+      } else {
+        log.warn("The job {} doesn't have an input for the attempt {}", input.getJobId(), input.getAttemptId());
```

Comment:
I think this should be "doesn't have any output" instead of "doesn't have an input"